istioctl/pkg/install/k8sversion/version.go

```diff
@@ -28,7 +28,7 @@ import (
 const (
 	// MinK8SVersion is the minimum k8s version required to run this version of Istio
 	// https://istio.io/docs/setup/platform-setup/
-	MinK8SVersion               = 19
+	MinK8SVersion               = 21
```

Comment:
We donot need to upgrade this unless it does not work in 1.19 or 1.20 anymore

Reply:
@hzxuzhonghu Shouldn't this be the value matches https://preliminary.istio.io/latest/docs/releases/supported-releases/#support-status-of-istio-releases, and https://github.com/istio/istio.io/blob/4827604b9c90e272d0d25bb173f79e19b2b57b3d/data/args.yml#L31-L32? I've checked the old PRs, and the min k8s version was always updated along with the Istio version.

Reply:
This is not to endusers, so i think not a must,  it is ok to be a superset of our announced supported versions.